generic_expression_patterns_modules/process.py

```diff
@@ -850,11 +822,7 @@ def get_shared_rank_scaled(
 
 
 def compare_gene_ranking(
```

Comment:
when comparing gene rankings, are higher ranks (more expressed genes) given more weight in the correlation calculation? I'm not sure if it makes sense to do that, I just thought of it based on your comment above (line 801).

Reply:
Our gene rank = median abs(log2FC) across multiple simulated experiments. So high gene rank corresponds to those genes that were found to be DE across multiple simulated experiments. 

So the correlation is showing how our gene rank compares with the ranking of the same genes found in the Crow et. al. paper, which performed DE across 600 experiments and ranked genes by how frequently they were found to be DE.

I will try to update the comments to explain this better